packages/theme-chalk/src/menu.scss

```diff
@@ -45,9 +45,12 @@
   }
 }
 
-@include b(menu) {
+:root {
+  // They are defined on :root so they can be inherited by sub-menu instead of overwritten
   @include set-component-css-var('menu', $menu);
```

Comment:
Do all the variables here need to be moved to `:root`? Is there a css variable that can still be placed in `el-menu`?

Reply:
1. All variables should be in one place to avoid from confusion.
2. All variables exist for styling, which shares the same problem.